app/src/main/java/com/kickstarter/services/mutations/CreateCheckoutData.kt

```diff
@@ -0,0 +1,7 @@
+package com.kickstarter.services.mutations
+
+import com.kickstarter.libs.RefTag
+import com.kickstarter.models.Project
+import com.kickstarter.models.Reward
+
+data class CreateCheckoutData(val project: Project, val amount: String, val locationId: String?, val rewardsIds: List<Reward>?, val refTag: RefTag?)
```

Comment:
Another nitpick here, 
reswarIds list should not be null ever, it generates an internal server error. It could should be an empty list when no values, so you can change it to `rewardsIds: List<Reward> = emptyList()`.

Here server response when you send null on that field:
![Screenshot 2024-02-01 at 9 08 21 AM](https://github.com/kickstarter/android-oss/assets/4083656/16592dce-9442-4b40-a1a9-a7b51083fb7a)

Here server response when you send empty list on that field:

![Screenshot 2024-02-01 at 9 02 55 AM](https://github.com/kickstarter/android-oss/assets/4083656/51da9795-e686-4d47-8871-7f2e4c3822e1)

Reply:
good point, it should also be an empty list if the use picks "no reward" so ill change that